src/chapter5/message-passing.md

```diff
@@ -1 +1,240 @@
 # Message Passing
+
+## Primary MPI Routines
+
+``` C
+int MPI_Init(int * argc, char ** argv);
+// initializes the MPI environment.
+//Argc argv are the parameters come
```

Comment:
Just make sure the C code style is consistent.

Reply:
Can I get you to elaborate on what you mean by this?